gateway/utils/syncman/syncman_crud.go

```diff
@@ -497,3 +498,37 @@ func (s *Manager) GetSchemas(ctx context.Context, project, dbAlias, col string)
 	}
 	return []interface{}{coll}, nil
 }
+
+// GetSecrets creates the config for the project
+// This function should be called only from setConfig method of any module
+func (s *Manager) GetSecrets(project, secretName, key string) (string, error) {
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	// Generate internal access token
+	token, err := s.adminMan.GetInternalAccessToken()
+	if err != nil {
+		return "", err
+	}
+
+	// makes http request to get secrets from runner
+	var vPtr map[string]interface{}
+	url := fmt.Sprintf("http://%s/v1/runner/%s/secrets?id=%s", s.runnerAddr, project, secretName)
+	if err := s.MakeHTTPRequest(ctx, "GET", url, token, "", map[string]interface{}{}, &vPtr); err != nil {
+		return "", err
+	}
+
+	v := vPtr["result"].([]interface{})
+	value := ""
+	for _, item := range v {
```

Comment:
no need to range over array as we are passing query param, 
we are assured that the length will be one, otherwise we will get error